scripts/setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
Some scripts to help make this easy to work with!

This one will install the `sqlx-cli` if needed, and then it also creates and migrates the Database

It DOES require the `DATABASE_URL` to be set and for Postgres to be installed